common/src/main/java/org/apache/drill/common/exceptions/UserException.java

```diff
@@ -536,6 +542,33 @@ public Builder pushContext(final String name, final double value) {
      * @return user exception
      */
     public UserException build(final Logger logger) {
+
+      // To allow for debugging:
+      //     A spinner code to make the execution stop here while the file '/tmp/drillspin' exists
```

Comment:
Would recommend `/tmp/drill/spin`. We already use `/tmp/drill` for other items, so this keep things tidy.

Reply:
Done